src/app/containers/app-header/app-header.component.ts

```diff
@@ -167,13 +175,12 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
 
   public getApplicationThemeForUser(applicationThemes: Theme[]): Theme {
 
-    const serialisedUserRoles: string = this.getSerialisedUserRolesFromCookie();
     const defaultTheme = this.getDefaultTheme();
 
-    if (serialisedUserRoles) {
+    if (this.serialisedUserRoles) {
       try {
-          const userRoles: string[] = this.deserialiseUserRoles(serialisedUserRoles);
-          return this.getUsersTheme(userRoles, applicationThemes, defaultTheme);
+          const userRolesDeserialised: string[] = this.deserialiseUserRoles(this.serialisedUserRoles);
```

Comment:
@paulhowes-HMCTS - any reason not to use `this.userRoles` here, since this is already assigned the value of `this.deserialiseUserRoles(this.serialisedUserRoles)`?

Reply:
Converting userRoles to a public property means this old code is no longer required so the method now uses this userRoles as per recommendation.